doc/content/source/kernels/ADMatCoupledDefectAnnihilation.md

```diff
@@ -0,0 +1,19 @@
+# ADMatCoupledDefectAnnihilation
+
+!syntax description /Kernels/ADMatCoupledDefectAnnihilation
+
+## Overview
+
+This class adds to the residual a contribution from
```

Comment:
```suggestion
This kernel object adds to the residual a contribution from
```

Prefer using the MOOSE naming convention instead of the C++ one here.